egs/aishell2/s5/local/prepare_eval_data.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# Copyright 2018 Emotech LTD (Author: Xuechen LIU)
+# Apache 2.0
+
+# This script downloads development and evaluation data and prepare necessary files.
+# Then it calls local/prepare_data.sh to prepare the data for testing.
+# To run this script from scratch, you may need to install unar as below:
+# apt install unar
+
+set -e
+. ./cmd.sh
+. ./path.sh
+
+if [ $# != 2 ]; then
+  echo "Usage: $0 <corpus-data-dir> <dict-dir>"
+  echo " $0 /export/AISHELL-2/iOS/data data/local/dict "
+  exit 1;
+fi
+ # note: the corpus directory shall be full name
+corpus=$1
+dict_dir=$2
+
+# check if the downloading and extracting has been done. If not, do it
+# note: 'evalT' stores the text files for each recording condition and
+# here we only employ iOS condition
+if [ ! -d $corpus/DEV ] || [ ! -d $corpus/TEST ] || [ ! -d $corpus/evalT ]; then
+  wget https://aishell-eval.oss-cn-beijing.aliyuncs.com/DEV.tar.gz -P $corpus && tar -xvzf $corpus/DEV.tar.gz -C $corpus || exit 1;
+  wget https://aishell-eval.oss-cn-beijing.aliyuncs.com/TEST.tar.gz -P $corpus && tar -xvzf $corpus/TEST.tar.gz -C $corpus || exit 1;
+  wget https://aishell-eval.oss-cn-beijing.aliyuncs.com/TEST.tar.gz -P $corpus && unar $corpus/evalT.rar $corpus || exit 1;
```

Comment:
typo here? should it be evalT.rar ?

Reply:
Just checked again from official website and it is evalT.rar unfortunately XD But we can make changes later if we can find a way change the archive name from the official source